services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/long_running_tasks.py

```diff
@@ -168,15 +168,28 @@ async def task_create_service_containers(
 
     assert shared_store.compose_spec  # nosec
 
-    async with event_propagation_disabled(app), _reset_on_error(shared_store):
+    async with event_propagation_disabled(app), _reset_on_error(
+        shared_store
+    ), ProgressBarData(
+        num_steps=5,
+        progress_report_cb=functools.partial(
+            post_progress_message,
+            app,
+            ProgressType.SERVICE_CONTAINERS_STARTING,
+        ),
+        description=IDStr("starting software"),
```

Comment:
This is strange. `IDStr` is not intended for description but rather string identifiers, note that ID means "identifier". Not sure why it was set a description as an identifier. Please check the other constrainted types already defined around IDStr and see which one might best fix.

Reply:
This one fits perfectly, We want to strip whitespaces and limit the size. Maybe a rename can be done in the future for an alias?